changelogs/fragments/429-add-cert-auth-variables.yml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - Add option to set the cert_auth_public_key and cert_auth_private_key parameters using the variables ansible_hashi_vault_cert_auth_public_key and ansible_hashi_vault_cert_auth_private_key (https://github.com/ansible-collections/community.hashi_vault/pull/429).
```

```suggestion
  - Add option to set the cert_auth_public_key and cert_auth_private_key parameters using the variables ansible_hashi_vault_cert_auth_public_key and ansible_hashi_vault_cert_auth_private_key (https://github.com/ansible-collections/community.hashi_vault/issues/428).
```
where possible we prefer to link to issues, or to both issue and PR

---

Thanks @briantist. I'll remember it for the next time :)

---

no worries, it is a very small thing and links to pull requests are also very common!